doc/content/getting-started/migrating/device-json.md

```diff
@@ -134,6 +136,11 @@ The linked specification is quite extensive, and contains a lot of fields that a
       "key":"01020304050607080102030405060708"
     }
   },
+  "mac_state":{
+    "current-parameters":{
+      "rx1_delay": "RX_DELAY_2"
+    }
+  },
```

Comment:
I do not know if this alone is sufficient, cc @rvolosatovs

Reply:
sufficient *for*?
The NS would treat suggested device RX1 Delay as follows:
- Device currently uses RX1 delay of 2, try to set it to 5. Next time device joins, use RX1 Delay 5.

Reply:
So this is the correct way to set the RX1 Delay to match the existing session, right? This PR makes sense?

Reply:
I think it makes more sense if you set `mac_settings.rx1_delay.value` to the same value or at least something other than 5, since that's the default and there's not much use from setting it.

Reply:
Then why do we have this PR at all? @rvolosatovs can you just suggest a change that you think is appropriate (i.e using the suggested changes, so we can see exactly what you would do.)